dev/com.ibm.websphere.appserver.features/visibility/public/batch-2.0/io.openliberty.batch-2.0.feature

```diff
@@ -0,0 +1,40 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.batch-2.0
+singleton=true
+visibility=public
+IBM-App-ForceRestart: uninstall, \
+ install
+IBM-API-Package: \
+  jakarta.batch.api; type="spec", \
+  jakarta.batch.api.chunk; type="spec", \
+  jakarta.batch.api.chunk.listener; type="spec", \
+  jakarta.batch.api.listener; type="spec", \
+  jakarta.batch.api.partition; type="spec", \
+  jakarta.batch.operations; type="spec", \
+  jakarta.batch.runtime; type="spec", \
+  jakarta.batch.runtime.context; type="spec", \
+  jakarta.inject;  type="spec"
+IBM-ShortName: batch-2.0
+Subsystem-Name: Jakarta Batch API 2.0
+-features=\
+  io.openliberty.jaxb-3.0, \
```

Comment:
Your original feature descriptor included feature `io.openliberty.jakarta.jaxb-3.0`.  Did you intend to change the included jaxb-3.0 feature?  I'm not sure this discrepancy is related to the MinJavaLevelTest failure, because I could not find jaxb-3.0 in the feature.java.levels.properties files.  These files seed the MinJavaLevelTest.  If you inadvertently changed the included feature, make the change and let's examine the next build.  Then either we fix the feature resolution error or we modify the test regarding batch-2.0.